resources/README.md

```diff
@@ -2,10 +2,10 @@
 
 ## Postgresql
 
-We will embed Postgres binaries to serve and store backend data. We will extract the binary to a binary path and use embedded-postgres to a Postgres server.
+We will embed Postgres binaries to serve and store backend data. We will extract the binary to a binary path and install Postgres. We will Go file suffix build tags to include the embedded file only for the build platform. For example, resources_darwin.go will only be included for building Bytebase on darwin platform. However, we will also use build tag `alpine` to differentiate builds on Linux and alpine Linux. We have to include alpine Linux because it's underlying libc is different from regular Linux, and we use alpine Linux for our docker images.
```

Comment:
```suggestion
We will embed Postgres binaries to serve and store backend data. We will extract the binary to a binary path and install Postgres. We will use Go file suffix build tags to include the embedded file only for the build platform. For example, resources_darwin.go will only be included for building Bytebase on darwin platform. However, we will also use build tag `alpine` to differentiate builds on Linux and alpine Linux. We have to include alpine Linux because its underlying libc is different from regular Linux, and we use alpine Linux for our docker images.
```